setup.py

```diff
@@ -4,6 +4,7 @@
 
 REQUIREMENTS = [
     'django-emailit',
+    'djangocms-text-ckeditor',
```

Comment:
So... Do I have to install the dependency even if I decide not to use the "contrib.email_notifications" app?